src/frontend/screens/Settings/components/Gamescope.tsx

```diff
@@ -37,6 +35,8 @@ const Gamescope = () => {
   )
 
   useEffect(() => {
+    if (!isLinux) return
```

Comment:
we were checking the location of the `gamescope` binary for windows and mac, unnecessary and the test helped identify this issue